scania-digital-design-system/tegel
# feat(message): prefix change

**Describe pull-request**  
Updated prefix for message component.

**Solving issue**  
Fixes: https://tegel.atlassian.net/browse/DTS-1640

**How to test**  
1. Go to storybook
2. Check in Components -> Message
3. Check the component API and style.

**Suggested test steps**
